kernel/component-manager/src/main/java/org/sakaiproject/modi/TomcatListener.java

```diff
@@ -0,0 +1,29 @@
+package org.sakaiproject.modi;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.catalina.Lifecycle;
+import org.apache.catalina.LifecycleEvent;
+import org.apache.catalina.LifecycleListener;
+
+/**
+ * Tomcat Bootstrapper for Sakai. Built as a LifecycleListener to be registered in server.xml, to start up all required
+ * components early, rather than waiting for them to be tripped by the first webapp.
+ * <p>
+ * Delegates all real work to the {@link Launcher}.
+ */
+@Slf4j
+public class TomcatListener implements LifecycleListener {
```

Comment:
I would like to propose that TomcatListener be renamed to StartSakaiListener as I think it will help people understand what it is for.

Reply:
Sure, I think this makes sense -- I'll update it.